programs/marginfi/src/instructions/marginfi_account/deposit.rs

```diff
@@ -48,9 +48,12 @@ pub fn lending_account_deposit<'info>(
 
     let mut bank = bank_loader.load_mut()?;
     let mut marginfi_account = marginfi_account_loader.load_mut()?;
+    let group = marginfi_group_loader.load()?;
```

Comment:
nit `group = marginfi` is inline with the other let statements but was previous `group = &marginfi` and we only ever use group as a reference.